[2025-06-02T15:41:02.511+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-02T15:41:03.328+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: log_monitoring_dag.create_output_folders manual__2025-06-02T15:37:47.618112+00:00 [queued]>
[2025-06-02T15:41:03.439+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: log_monitoring_dag.create_output_folders manual__2025-06-02T15:37:47.618112+00:00 [queued]>
[2025-06-02T15:41:03.440+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-06-02T15:41:04.109+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): create_output_folders> on 2025-06-02 15:37:47.618112+00:00
[2025-06-02T15:41:04.142+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'log_monitoring_dag', 'create_output_folders', 'manual__2025-06-02T15:37:47.618112+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/log_monitoring_dag.py', '--cfg-path', '/tmp/tmphn5u7i4m']
[2025-06-02T15:41:04.145+0000] {standard_task_runner.py:105} INFO - Job 116: Subtask create_output_folders
[2025-06-02T15:41:04.147+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1552) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-02T15:41:04.148+0000] {standard_task_runner.py:72} INFO - Started process 1573 to run task
[2025-06-02T15:41:04.529+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-02T15:41:04.639+0000] {task_command.py:467} INFO - Running <TaskInstance: log_monitoring_dag.create_output_folders manual__2025-06-02T15:37:47.618112+00:00 [running]> on host c47ce43ea693
[2025-06-02T15:41:06.575+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='log_monitoring_dag' AIRFLOW_CTX_TASK_ID='create_output_folders' AIRFLOW_CTX_EXECUTION_DATE='2025-06-02T15:37:47.618112+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-02T15:37:47.618112+00:00'
[2025-06-02T15:41:06.601+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-02T15:41:08.249+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-02T15:41:08.392+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p /opt/***/app/output/error_logs']
[2025-06-02T15:41:08.405+0000] {subprocess.py:86} INFO - Output:
[2025-06-02T15:41:09.215+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-02T15:41:09.401+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-02T15:41:09.401+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=log_monitoring_dag, task_id=create_output_folders, run_id=manual__2025-06-02T15:37:47.618112+00:00, execution_date=20250602T153747, start_date=20250602T154103, end_date=20250602T154109
[2025-06-02T15:41:10.035+0000] {job.py:229} INFO - Heartbeat recovered after 10.52 seconds
[2025-06-02T15:41:10.045+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-02T15:41:10.359+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-02T15:41:10.364+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
